go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
-	github.com/go-openapi/spec v0.20.3
+	github.com/go-openapi/spec v0.19.5
```

Comment:
this is going backwards @andyzhangx . 

Also i don't see anything related to the azure deps :)